deploy/backplane/lpsre/20-lpsre-addon-operator.SubjectPermission.yml

```
@@ -1,18 +1,18 @@
 apiVersion: managed.openshift.io/v1alpha1
 kind: SubjectPermission
 metadata:
-  name: backplane-lpsre-addon-operator-admins
+  name: backplane-lpsre-addon-operator
   namespace: openshift-rbac-permissions
 spec:
   clusterPermissions:
-  - backplane-lpsre-addon-operator-admin
+  - backplane-lpsre-addon-operator-cluster
   - backplane-readers-cluster
   permissions:
   - clusterRoleName: admin
     namespacesAllowedRegex: (^openshift-addon-operator$)
-  - clusterRoleName: backplane-lpsre-monitoring
+  - clusterRoleName: backplane-lpsre-monitoring-project-project
```

```suggestion
  - clusterRoleName: backplane-lpsre-monitoring-project
```

Please make sure referenced role names are valid.